docs/reference/method/MongoDBDatabase-aggregate.txt

```diff
@@ -0,0 +1,71 @@
+==============================
+MongoDB\\Database::aggregate()
+==============================
+
+.. versionadded:: 1.5
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Database::aggregate()
+
+   Executes an :manual:`aggregation framework pipeline
+   </core/aggregation-pipeline>` operation on the collection.
+
+   .. code-block:: php
+
+      function aggregate(array $pipeline, array $options = []): Traversable
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBDatabase-method-aggregate-param.rst
+
+   The ``$options`` parameter supports the following options:
+
+   .. include:: /includes/apiargs/MongoDBDatabase-method-aggregate-option.rst
+
+Return Values
+-------------
+
+A :php:`MongoDB\\Driver\\Cursor <class.mongodb-driver-cursor>` or
+:php:`ArrayIterator <arrayiterator>` object. In both cases, the return value
+will be :php:`Traversable <traversable>`.
+
+Errors/Exceptions
+-----------------
+
+.. include:: /includes/extracts/error-unexpectedvalueexception.rst
+.. include:: /includes/extracts/error-unsupportedexception.rst
+.. include:: /includes/extracts/error-invalidargumentexception.rst
+.. include:: /includes/extracts/error-driver-runtimeexception.rst
+
+.. _php-agg-method-behavior:
+
+Behavior
```

Comment:
I think we can drop this section entirely, since we're also pulling `useCursor` documentation from this method. I'm not even sure if the admin/diagnostic pipeline operators are compatible with `explain`, which this text would technically explain, but it's such an edge case I think we'd do better to remove this entirely lest we risk confusing readers.

Reply:
> I'm not even sure if the admin/diagnostic pipeline operators are compatible with `explain`

I had to check myself, but it's perfectly fine to use `explain` with diagnostic pipeline operators.
```
> db.aggregate([{$currentOp: {}}], {explain: true})
{ "stages" : [ { "$currentOp" : { } } ], "ok" : 1 }
```

Removing the block sounds good for now. It might be worth explaining the different behaviour when using `explain`, but mentioning `useCursor` is definitely confusing.